package.json

```diff
@@ -32,14 +32,12 @@
   },
   "dependencies": {
     "call-bind": "^1.0.2",
-    "es-get-iterator": "^1.1.3",
     "get-intrinsic": "^1.2.0",
     "is-arguments": "^1.1.1",
     "is-array-buffer": "^3.0.1",
     "is-date-object": "^1.0.5",
     "is-regex": "^1.1.4",
     "is-shared-array-buffer": "^1.0.2",
-    "isarray": "^2.0.5",
```

Comment:
as is this

Reply:
Hi Jordan, thanks for your fast response (and for being polite to discuss instead of just closing the pull request 😊). I am doing research on finding unused dependencies for NodeJs packages. I will check the correctness by removing the dependencies from package.json, performing the npm install, and running the package's test. 

I understand that there are statements in files showing the dependencies are "used"; for example, they are required in some files. I would like to know if the dependencies are really necessary, why the package still manages to pass the tests after we remove them from package.json and do not install them at all. 

How are they necessary for your package? If they are, why do the tests not test them? If not, why do we still need them? 

Look forward to your opinions. 😊

Reply:
Yes, they are necessary - when you remove them from package.json, they're likely still installed and hoisted because other things in the graph use them - if you `rm -rf node_modules/isarray`, for example, the tests will fail.

In other words, you can't simply remove things from package.json to determine if a dependency is unused. Presence in package.json has *many* effects that don't exclusively include "causing the package to be available".

Reply:
Exactly! I highly agree.